# The binary is not installed by default which caused the sanity check to fail
# @author Stefan Wolfsheimer, SURF

--- libaec-v1.1.4.orig/packaging/CMakeLists.txt	2025-06-12 13:06:42.000000000 +0200
+++ libaec-v1.1.4/packaging/CMakeLists.txt	2025-08-07 14:58:58.917432324 +0200
@@ -15,6 +15,9 @@
   "${CMAKE_CURRENT_BINARY_DIR}/libaec-config-version.cmake"
   DESTINATION "${libaec_INSTALL_CMAKEDIR}")
 
+install(TARGETS graec
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+
 set(TARGETS_EXPORT)
 if(BUILD_SHARED_LIBS)
   set(TARGETS_EXPORT aec-shared-objects sz-shared-objects aec-shared sz-shared)
